# üî• News
SIGEL is published in Genome Biology! (Sep. 2025)

# üîé Researchs
I am currently working on cross-sample differential expression analysis in spatial transcriptomics.

# üìù Publications 

<div class='paper-box'><div class='paper-box-image'><div><div class="badge">Genome Biology 2025</div><img src='images/Figure_1_GB_00.png' alt="sym" width="100%"></div></div>
<div class='paper-box-text' markdown="1">

[SIGEL: a context-aware genomic representation learning framework for spatial genomics analysis](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-025-03748-7) \\
**Wenlin Li#**, Maocheng Zhu, Yucheng Xu, Mengqian Huang, Jin Chen, Hao Wu*, Xiaobo Sun*

- We develop a ‚Äúcontext-aware, self-supervised Spatially Informed Gene Emedding Learning (SIGEL)‚Äù framework, addressing the gap in methods for generating gene manifolds and utilizing spatial genomic context.
- SIGEL generates gene representations (SGRs) that are context-aware, semantically rich, and robust against technical artifacts across different samples, enhancing the biological relevance and functional understanding of genomic contexts.
- SGRs are applied to important biomedical research tasks such as imputing missing genes in FISH-based ST, detecting genes with specific spatial expression patterns, identifying disease-associated genes and gene-gene interactions, and improving spatial clustering.
- Extensive experiments with diverse ST datasets across various platforms, species, and tissues demonstrate the superior performance of SGRs in facilitating various analytical tasks, affirming their potential in data-driven genomic research.
</div>
</div>


<div class='paper-box'><div class='paper-box-image'><div><div class="badge">ACM-MM 2024</div><img src='images/DARLC.png' alt="sym" width="100%"></div></div>
<div class='paper-box-text' markdown="1">

[Dual Advancement of Representation Learning and Clustering for Sparse and Noisy Images](https://arxiv.org/abs/2409.01781) \\
**Wenlin Li#**, Yucheng Xu#, Xiaoqing Zheng, Suoya Han, Jun Wang, Xiaobo Sun*

- DARLC is the first unified framework for joint representation learning and clustering of sparse and noisy images (SNIs).
- DARLC integrates contrastive learning, masked image modeling, and a graph attention-based augmentation to enhance representation learning.
- DARLC employs a Student‚Äôs t mixture model for robust SNI clustering, addressing class collision and improving adaptability.
- DARLC outperforms state-of-the-art methods on 12 SNI datasets in clustering quality and downstream task performance.
</div>
</div>

<div class='paper-box'><div class='paper-box-image'><div><div class="badge">Genome Biology (revising) 2025</div><img src='images/Figure_1_GB_00.png' alt="sym" width="100%"></div></div>
<div class='paper-box-text' markdown="1">


[MEATRD: Multimodal Anomalous Tissue Region Detection Enhanced with Spatial Transcriptomics](https://ojs.aaai.org/index.php/AAAI/article/view/33409) \\
Kaicheng Xu#, Qilong Wu#, Yan Lu, Yinan Zheng, **Wenlin Li**, Xingjie Tang, Jun Wang, Xiaobo Sun*

- MEATRD is the first method to integrate histology images and ST data for anomalous tissue region (ATR) detection across tissue regions.
- MEATRD leverages a masked graph dual-attention transformer to enable cross-modality information sharing and mitigate over-generalization.
- MEATRD combines reconstruction-based learning with one-class classification to detect ATRs with minimal visual differences from normal tissues.
- MEATRD outperforms state-of-the-art methods across eight ST datasets, demonstrating superior ATR detection performance.
</div>
</div>
