# üîé Researchs


# üìù Publications 


<div class='paper-box'><div class='paper-box-image'><div><div class="badge">ACM-MM 2024</div><img src='images/DARLC.png' alt="sym" width="100%"></div></div>
<div class='paper-box-text' markdown="1">

[Dual Advancement of Representation Learning and Clustering for Sparse and Noisy Images](https://arxiv.org/abs/2409.01781) \\
**Wenlin Li#**, Yucheng Xu#, Xiaoqing Zheng, Suoya Han, Jun Wang, Xiaobo Sun*

- DARLC is the first unified framework for joint representation learning and clustering of sparse and noisy images (SNIs).
- DARLC integrates contrastive learning, masked image modeling, and a graph attention-based augmentation to enhance representation learning.
- DARLC employs a Student‚Äôs t mixture model for robust SNI clustering, addressing class collision and improving adaptability.
- DARLC outperforms state-of-the-art methods on 12 SNI datasets in clustering quality and downstream task performance.
</div>
</div>

<div class='paper-box'><div class='paper-box-image'><div><div class="badge">Preprint</div><img src='images/CD3.png' alt="sym" width="100%"></div></div>
<div class='paper-box-text' markdown="1">

Cross-sample Detection of Differential Expression Genes and Domains in Spatial Transcriptomics\\
**Wenlin Li#**, Yan Lu#, Maocheng Zhu, Hao Wu, Xiaobo Sun*

- CD3 is the first model for DEG detection in ST samples across different conditions.
- CD3 maps gene expression into a unified latent space, identifies common genes via Fisher‚Äôs test, and determines DEGs through ANOVA.
- CD3 identifies DEGs and DEDs, quantifies developmental/recovery processes, detects vulnerable regions, and analyzes spot-level pathway enrichment.
- CD3 has shown outstanding performance across various diseases, developmental and injury contexts, different sequencing technologies, and even cross-modal data.
</div>
</div>

<div class='paper-box'><div class='paper-box-image'><div><div class="badge">Preprint</div><img src='images/SPIDER.png' alt="sym" width="100%"></div></div>
<div class='paper-box-text' markdown="1">

Detecting biologically significant spatial gene expression patterns in spatial transcriptomics with SPIDER \\
**Wenlin Li**, Mengqian Huang#, Ye Qin#, Yucheng Xu, Hao Wu, Xiaobo Sun*


- SPIDER is the first framework to unify the detection of SCGs, SVGs, and SPGs in ST data.
- SPIDER leverages self-supervised learning to extract semantically enriched gene representations from spatial genomic contexts.
- SPIDER identifies biologically meaningful gene groups, ranks SVGs effectively, and detects disease-relevant expression patterns.
- SPIDER outperforms state-of-the-art methods across diverse species, tissues, diseases, and ST datasets.
</div>
</div>

<div class='paper-box'><div class='paper-box-image'><div><div class="badge">AAAI 2024</div><img src='images/MEATRD.png' alt="sym" width="100%"></div></div>
<div class='paper-box-text' markdown="1">

MEATRD: Multimodal Anomalous Tissue Region Detection Enhanced with Spatial Transcriptomics \\
Kaicheng Xu#, Qilong Wu#, Yan Lu, Yinan Zheng, **Wenlin Li**, Xingjie Tang, Jun Wang, Xiaobo Sun*

- MEATRD is the first method to integrate histology images and ST data for anomalous tissue region (ATR) detection across tissue regions.
- MEATRD leverages a masked graph dual-attention transformer to enable cross-modality information sharing and mitigate over-generalization.
- MEATRD combines reconstruction-based learning with one-class classification to detect ATRs with minimal visual differences from normal tissues.
- MEATRD outperforms state-of-the-art methods across eight ST datasets, demonstrating superior ATR detection performance.
</div>
</div>
